### PR TITLE
Redesigning welcome page

### LINKED ISSUE #447: Redesigning Welcome Page
### Description
Please leave a brief description of the bug or feature request:
Not sure where to post my designs, so I am posting it both on Gitter and here. Please review and suggest improvements, if any.
Wokbench-Welcome

### PR DESCRIPTION
**Pull request by Vishal-raj-1**
## Description

I have  redesign the welcome page. It will fixes #447 .

## Preview 

preiew

@majurg Please review it !!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -118,26 +118,30 @@
...
+                                <i class="fa fa-floppy-o"></i> 
+                                <h4>Save SQLite File</h4>
+                            </div>
+                        </div>
+                        <div class="col">
+                            <h3>Quick Links: </h3>
+                            <div class="btn-group-horizontal" role="group" aria-label="...">
+                                <a href="https://github.com/nexB/scancode-workbench/" class="btn btn-lg btn-default">GitHub Repository</a>
+                                <a href="https://scancode-workbench.readthedocs.io/" class="btn btn-lg btn-default">Documentation</a>
+                                <a href="https://github.com/nexB/scancode-workbench/issues" class="btn btn-lg btn-default">Report a Bug or Request a Feature</a>
```
Comment (steven-esser) [inline index.html]: I think this section needs to be centered, similar to how it looks in #447

File: index.html [hunk 2]
```diff
@@ -118,26 +118,30 @@
             <!-- Welcome Page -->  
             <div class="tab-pane container" id="tab-welcomepage">
                 <div id="welcomepage-container">
-                    <div id="wecomepage-title">
+                    <div id="welcomepage-title">
```
Comment (steven-esser) [inline index.html:121]: Why does this need to be changed?

**Review by steven-esser — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (steven-esser) [reply index.html:121]: Oh I see now, its a typo :)

File: index.html (hunk 1 above)
Comment (Vishal-raj-1) [reply index.html]: Sure !!

**Comment by Vishal-raj-1** [comment]
## Preview

preview2


@majurg Now I have update . Sir, You can check the preview.

**Review by steven-esser — CHANGES_REQUESTED** [review]
This is slightly better, but the buttons are not centered like the large icons above.

**Comment by Vishal-raj-1** [comment]
preview3


@majurg is this look good ?

**Review by steven-esser — CHANGES_REQUESTED** [review]
@Vishal-raj-1 This looks great now, thanks!

Two things left before I can merge:
1) Please squash your commits on this PR to single one. 
2) Please address the lint errors found here:
```
1) ESLint

       validates /home/travis/build/nexB/scancode-workbench/assets/app/js/controllers/conclusionDialog.js:

     AssertionError: 210:23 Strings must use singlequote - quotes

232:23 Strings must use singlequote - quotes

253:23 Strings must use singlequote - quotes

275:23 Strings must use singlequote - quotes

309:23 Strings must use singlequote - quotes

334:23 Strings must use singlequote - quotes

361:23 Strings must use singlequote - quotes

382:23 Strings must use singlequote - quotes

      at Context.it (test/eslint.test.js:41:14)
```

You can verify linting prior to pushing via running `npm test`

**Comment by Vishal-raj-1** [comment]
@majurg Sir I have resolved the Lint error. Please review it !!

**Review by steven-esser — APPROVED** [review]

**Comment by Vishal-raj-1** [comment]
Thanks for merging @majurg sir !!

**Comment by nitin10s** [comment]
This looks awesome, sorry I was not active due to some personal reasons. Shall I send the icons that I've used in the design?

**Comment by Vishal-raj-1** [comment]
> This looks awesome, sorry I was not active due to some personal reasons. Shall I send the icons that I've used in the design?

Yes sir !! I will replace this with your icons. Which looks cools to me !!

**Pull request merged**